nixos/modules/services/databases/couchdb.nix

```diff
@@ -157,12 +157,15 @@ in {
         mkdir -p ${cfg.databaseDir};
         mkdir -p ${cfg.viewIndexDir};
         touch ${cfg.configFile}
+        touch ${cfg.logFile}
```

Comment:
Touching log files is not a good idea for analyzing problems in software. E.g. from an `ls -l` you may want to see when the log file was last written. Can you use `touch -a`  so that it doesn't change the mod time?